portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -126,4 +131,22 @@ private String getBody(HttpServletRequest request) throws IOException {
 
         return "";
     }
+
+    private double getSentiment(String comment) throws IOException {
```

Comment:
[nit] I would rename this to something like "analyzeSentiment", the name collision with the Comment class' getSentiment was a bit confusing.